pkg/virtctl/vmexport/vmexport.go

```diff
@@ -166,8 +192,13 @@ func usage() string {
 	{{ProgramName}} vmexport download vm1-export --volume=volume1 --output=disk.img.gz
   
 	# Create a VirtualMachineExport and download the requested volume from it
-	{{ProgramName}} vmexport download vm1-export --vm=vm1 --volume=volume1 --output=disk.img.gz`
+	{{ProgramName}} vmexport download vm1-export --vm=vm1 --volume=volume1 --output=disk.img.gz
 
+	# Create a VirtualMachineExport and get the VirtualMachine manifest in Yaml format
+	{{ProgramName}} vmexport manifest vm1-export --vm=vm1
+
+	# Get the VirtualMachine manifest in Yaml format from an existing VirtualMachineExport including CDI header secret
+	{{ProgramName}} vmexport manifest existing-export --include-secret`
 	return usage
```

Comment:
How about an example with export-url?

Reply:
I want to improve the export-url in another PR to include a `--port-forward` flag, that sets up the port-forward for you. Until then I don't want to promote the export-url flag